### PR TITLE
Output multiple files

### PR DESCRIPTION
**Pull request by bfirsh**
Lots of models output multiple files -- either progressive output or batches of files.

This pull request makes `cog predict` output them as `output.0.png`, `output.1.png`, etc. The test is pretty illustrative.

### CODE REVIEW

File: pkg/cli/predict.go [hunk 1]
```diff
@@ -195,17 +207,39 @@ func predictIndividualInputs(predictor predict.Predictor, inputFlags []string, o
...
+
+	for i, output := range outputs {
+		outputString := output.(string)
+		dataurlObj, err := dataurl.DecodeString(outputString)
+		if err != nil {
+			return fmt.Errorf("Failed to decode dataurl: %w", err)
+		}
+		out := dataurlObj.Data
+		extension := mime.ExtensionByType(dataurlObj.ContentType())
+		outputPath := fmt.Sprintf("output.%d%s", i, extension)
```
Comment (zeke) [inline pkg/cli/predict.go:234]: Do we wanna put some leading zeros on this file format so the files will sort alphabetically beyond 9? `output.000.png`

**Review by zeke — APPROVED** [review]
I took the liberty of adding some docs.

Inline question about leading zeros on the file format, but otherwise this looks good to go.

File: docs/python.md [hunk 2]
```diff
@@ -145,6 +150,27 @@ class Predictor(BasePredictor):
...
+        output = []
+        for i, prediction in enumerate(predictions):
+            out_path = Path(f"/tmp/out-{i}.txt")
+            with out_path.open("w") as f:
+                f.write(prediction)
+            output.append(out_path)
+        return output
+```
+
+Files are named in the format `output.<index>.<extension>`, e.g. `output.0.txt`, `output.1.txt`, and `output.2.txt` from the example above.
```
Comment (bfirsh) [inline docs/python.md:172]: Not a blcoker to shipping, but this is a bit out of context -- nothing mentions `cog predict` anywhere.

Comment (bfirsh) [reply docs/python.md:172]: Maybe this should live in the CLI documentation, but is currently just `cog predict --help`.

Comment (zeke) [reply docs/python.md:172]: Ah yeah good point. Should we take this last line out and add a TODO on https://github.com/replicate/cog/issues/433?

**Pull request merged**